spec/lib/roadie/net_http_provider_spec.rb

```diff
@@ -43,6 +43,21 @@ module Roadie
       }.to_not raise_error
     end
 
+    describe "encoding" do
+      fake_body = "p { color: green; }"
+      it "should respect charset header if supplied" do
+        stub_request(:get, url).and_return(body: fake_body, headers: { "Content-Type" => "text/html;charset=ISO-8859-1" })
+        expect(Stylesheet).to receive(:new).with(url, fake_body.force_encoding("ISO-8859-1")).and_call_original
```

Comment:
`force_encoding` actually mutates the `fake_body` string, changing it between examples. That also means, AFAIK, that this test will always pass, even if encoding was never adjusted in the implementation code.

I'll adjust this after I merge. :-)